.ci/pitest-suppressions/pitest-imports-suppressions.xml

```diff
@@ -1,12 +1,102 @@
+    <description>Removed assignment to member variable customImportOrderRules</description>
+    <lineContent>customImportOrderRules = inputCustomImportOrder;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>FileImportControl.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.FileImportControl</mutatedClass>
+    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable patternForExactMatch</description>
+    <lineContent>patternForExactMatch = null;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>FileImportControl.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.FileImportControl</mutatedClass>
+    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.ArgumentPropagationMutator</mutator>
+    <description>replaced call to com/puppycrawl/tools/checkstyle/checks/imports/FileImportControl::encloseInGroup with argument</description>
+    <lineContent>this.name = encloseInGroup(name);</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>ImportControlCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.ImportControlCheck</mutatedClass>
+    <mutatedMethod>beginTree</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable currentImportControl</description>
+    <lineContent>currentImportControl = null;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>ImportControlLoader.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.ImportControlLoader</mutatedClass>
+    <mutatedMethod>load</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
+    <description>removed call to java/lang/Exception::getMessage</description>
+    <lineContent>+ &quot; - &quot; + ex.getMessage(), ex);</lineContent>
+  </mutation>
+
   <mutation unstable="false">
     <sourceFile>ImportControlLoader.java</sourceFile>
     <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.ImportControlLoader</mutatedClass>
     <mutatedMethod>startElement</mutatedMethod>
     <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_ELSE</mutator>
     <description>removed conditional - replaced equality check with false</description>
-    <lineContent>else if (ALLOW_ELEMENT_NAME.equals(qName) || "disallow".equals(qName)) {</lineContent>
+    <lineContent>else if (ALLOW_ELEMENT_NAME.equals(qName) || &quot;disallow&quot;.equals(qName)) {</lineContent>
```

Comment:
@Vyom-Yadav was this an improvement to your script?

Reply:
Yes, `lineContent = XmlUtil.escapeXml(file.readLines().get(lineNumber - 1).trim())`, it was there earlier also, but this script uses latest version of groovy (as its executed in local) so it might be due to that.